app/views/track_stream/index.erb

```diff
@@ -20,4 +20,7 @@
       }
     %>
   </div>
+
+  <%= erb :"gitter/chat_button", locals: { control: stream } %>
```

Comment:
I think that we should always pass a track, which will have the track ID, the language name, and the `gitter` value.